### ISSUE TITLE
Document add custom colormap to Geo2Grid IR channel

### ISSUE THREAD

**Issue by djhoese** [ssec/polar2grid#647]
In an effort to not make users upset, we don't want to change the default enhancement for all IR channels by adding colormaps. Instead, we plan on documenting how to add a custom colormap via a custom enhancement configuration in the PDF documentation and website. This could/should include mentioning replacing/overwriting the existing enhancement (meaning "C07" is still called "C07") and also adding a new "single band" composite that adds this colormapped version under a new name (ex. "C07_color").

**Comment by graemely** [comment]
Yes, there would be a C??_color for every IR channel, but I was thinking it would only be listed in `--list-products-all` (not `--list-products`). Is there a reason to want to keep the list of non-default products short?

**Comment by djhoese** [comment]
Hhhmm I think they'd be considered and listed as "custom user products" if they aren't considered "supported G2G products" and that's with the plain `--list-products` option too.

For keeping the list short: user confusion is probably the biggest one.

**Comment by graemely** [comment]
"Custom user products" sounds to me like something the user created. 

But regardless, I would argue we should plan to continue adding products that we think will have value for users and that have been vetted in some way. Maybe it should be a requirement that there is some documentation that we can link users to.

**Comment by djhoese** [comment]
Oh I wasn't saying the custom user products was a good thing. I just meant (without saying) that there isn't a way to add these so that they show up only in the "all" list.

What documentation do you mean? Like a NOAA/NASA guide about a product?

**Comment by graemely** [comment]
Yes, something like a NOAA/NASA guide, or a paper. Even a set of slides would be better than nothing. We can host those kinds of things on the CSPP Geo website if needed.

**Comment by djhoese** [comment]
That might be hard for the colorized IR images as those are "it's what AWIPS does" and in some cases it sounds like TimS would prefer if we didn't follow the AWIPS default as he "lost the battle" on what colormap to be the default. I think a lot of them come down "oh this looks prettier than that one".

**Comment by graemely** [comment]
My own criteria may be close to "oh this looks prettier than that one", but I'm confident Tim has a good sciency reason for his preferences. As has been noted we can't make everyone happy, so I think the best we can do is gather input and make an informed decision. Suggest asking him why he prefers that flavor, and what documentation exists.

**Comment by djhoese** [comment]
I think documenting and closing this issue would also qualify https://github.com/ssec/polar2grid/issues/286 for closing.
